README.md

```diff
@@ -1 +1,17 @@
-# Android-Maps-Demo
+Bottle Rocket Android Maps Guidelines
+=====================================
+
+## Steps on obtaining an API key to run the Android Maps Demo.
+1. Get the API Key at https://mapsplatform.google.com and click on get started. (You might need to setup billing).
+2. In the console click on the left hamburger menu on the top-most left hand side of the site and choose ```API & Services``` > ```credentials```.
```

Comment:
@Josh-Gebbeken nit: Prefer wrapping a few words or a single line of code using with a single backtick at the start and end `like this` (see https://www.markdownguide.org/basic-syntax/#code). Use the triple backticks to delineate multiline code blocks (see https://www.markdownguide.org/extended-syntax/#fenced-code-blocks).

Replace all triple backtick wrappers with single backtick wrappers.